app/services/project.ts

```diff
@@ -0,0 +1,88 @@
+import Service from '@ember/service';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency';
+import Store from '@ember-data/store';
+import IntlService from 'ember-intl/services/intl';
+
+// eslint-disable-next-line ember/use-ember-data-rfc-395-imports
+import DS from 'ember-data';
+
+import ProjectModel from 'irene/models/project';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+
+type ProjectQueryResponse = DS.AdapterPopulatedRecordArray<ProjectModel> & {
+  meta?: { count: number };
+};
+
+export const DEFAULT_PARAMS = {
```

Comment:
Can we have a better name for this to correlate with the project list query?

Reply:
done